src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/components/enterprise-alert/index.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
```

Comment:
💭 wondering if this is the right place for this to live. I also thought of putting it in 'author-primitives'?

Reply:
Personally I think we should consolidate `author-primitives` into `dev-dot-content/mdx-components`. I see them like "legacy" components since they were primarily kept separate for the "lift & shift" of .io sites to `dev-portal`.